appveyor.yml

```diff
@@ -111,7 +111,7 @@
                 $env:BOOST_LIBRARYDIR = "C:/Libraries/boost_1_${env:BOND_BOOST}_0/lib${env:BOND_ARCH}-msvc-${env:BOND_VS_NUM}.0"
             }
 
-            choco install haskellplatform -version 2014.2.0.0 -y
+            choco install haskell-stack -y
 
             # choco install updated the path, so re-read them from the registry and reset $env:path
```

Comment:
If this doesn't include version details, we should print which version of stack got used for debugging.